businessRules/users.js

```diff
@@ -4,6 +4,13 @@ const { createUser } = require('../models/users');
 const { getUsers, setUsers } = require('../utils/storage');
 
 function registerUser(userName, password, firstName, lastName) {
+    if (!userName) {
+        throw new Error('Error. No Username provided');
```

Comment:
Inconsistent error message format. 

`Error. No Username provided` vs `No such user.`

Choose a single error format and follow it across the entire codebase. Code consistency is important.

Reply:
The thing is, that "Error. There was no username provided." is being used when !userName
"No such user" I'm using when the user.id during the .find  method doesn't match user.id